### ISSUE TITLE
Wrong event.keyCode on keydown event for Korean/Japanese input

### ISSUE THREAD

**Issue by Kid-Binary** [atom/atom#12682]
### Prerequisites

* [x] Can you reproduce the problem in [safe mode](http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode)?
* [x] Are you running the [latest version of Atom](http://flight-manual.atom.io/hacking-atom/sections/debugging/#update-to-the-latest-version)?
* [x] Did you check the [debugging guide](http://flight-manual.atom.io/hacking-atom/sections/debugging/)?
* [x] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
* [x] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
* [x] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if your bug or enhancement is already reported?

### Description

Atom does not detects a correct `keyCode` of keydown event for Korean and Japanese keyboard input.
(when user input is set to **Hangul** (Korean) or **Anthy** (Japanese) keyboard input).

Any other standard keyboard layout input gives desired key code (`q = 81`, etc.), but these input modes always posting **229** as event `keyCode`. This is Atom event subscriber I use:

```javascript
editorView.addEventListener('keydown', (event) => {
    console.log(event.keyCode);
});
```

And it works perfect for other keyboard layouts.

Native JS in HTML test page (out of Atom):

```javascript
document.onkeydown = function(event) {
    console.log(event.keyCode);
};
```

In Chrome browser it gives the correct `keyCode` even for Korean and Japanese input. So issue is Atom-specific.

**Expected behavior:** `keyCode` for `keydown` event on `q` should be equal to **81** (example).

**Actual behavior:** `keyCode` for `keydown` event on `q` equals to **229** (like any other keys).

### Versions

Atom 1.10.0
Ubuntu 16.04 x64

**Comment by 50Wliu** [comment]
If it's fixed on Chrome, I'm going to assume that this will be fixed in Atom 1.12.0 when it's released as we've recently upgraded to a more recent version of Chrome.

**Comment by Kid-Binary** [comment]
Still, Atom 1.12.0 release did not fix the described issue. Versions:
```
Atom    : 1.12.7
Electron: 1.3.13
Chrome  : 52.0.2743.82
Node    : 6.5.0
```
Console logs **229** on `keydown` in Atom (on any character key), while Chrome console shows the correct values.
